docs/client/rest/rest_api.md

```diff
@@ -449,13 +454,15 @@ Refresh the Hadoop configurations of the Kyuubi server.
 
 ### POST /admin/refresh/user_defaults_conf
 
-Refresh the [user defaults configs](../../configuration/settings.html#user-defaults) with key in format in the form of `___{username}___.{config key}` from default property file.
+Refresh the [user defaults configs](../../configuration/settings.html#user-defaults) with key in format in the form
+of `___{username}___.{config key}` from default property file.
 
 ### POST /admin/refresh/kubernetes_conf
 
 Refresh the kubernetes configs with key prefixed with `kyuubi.kubernetes` from default property file.
 
-It is helpful if you need to support multiple kubernetes contexts and namespaces, see [KYUUBI #4843](https://github.com/apache/kyuubi/issues/4843).
+It is helpful if you need to support multiple kubernetes contexts and namespaces,
+see [KYUUBI #4843](https://github.com/apache/kyuubi/issues/4843).
```

Comment:
please revert unnecessary docs format change in several places

Reply:
It may be caused by my using the mvn spotless:apply command, I will fix it immediately.